lib/features/login/presentation/login_bindings.dart

```diff
@@ -54,7 +54,7 @@ class LoginBindings extends BaseBindings {
 
   @override
   void bindingsController() {
-    Get.lazyPut(() => LoginController(
+    Get.create(() => LoginController(
```

Comment:
We use `create` instead of `lazyPut` in order to always create a new `instance` every time we move to the `Login` screen. If you use `lazyPut` it has only 1 instance, so in case when you do the `Login -> Session -> Login` screen move. If this is done very quickly, it will result that LoginView2 will use the controller of LoginView1 while LoginController1 has been disposed.